src/geoapis/vector.py

```diff
@@ -217,3 +222,29 @@ def get_features(self, layer: int):
             features = None
 
         return features
+
+
+class Linz(WfsQueryBase):
+    """ A class to manage fetching Vector data from LINZ.
+
+    API details at: https://www.linz.govt.nz/data/linz-data-service/guides-and-documentation/wfs-spatial-filtering
+
+    Note that depending on the layer the geometry_name may vary. The LINZ LDS uses 'shape' for most property/titles, and
+    GEOMETRY for most other layers including Hydrographic and Topographic data.
+    """
+
+    NETLOC_API = "data.linz.govt.nz"
+    GEOMETRY_NAMES = ['GEOMETRY', 'shape']
+
+
+class Lris(WfsQueryBase):
+    """ A class to manage fetching Vector data from LRIS.
+
+    API details at: https://lris.scinfo.org.nz/p/api-support-wfs/ z
+
+    Note that depending on the layer the geometry_name may vary. The LRIS appears to follow the LINZ LDS uses of 'shape'
+    for most property/titles, and GEOMETRY for most other layers including Hydrographic and Topographic data.
+    """
+
+    NETLOC_API = "lris.scinfo.org.nz"
+    GEOMETRY_NAMES = ['GEOMETRY', 'Shape']
```

Comment:
The other option rather than inheritance would be composition. ie accepting these values in the constructor of the base class rather than child classes. That's a discussion point. I like this way you've done but would like to talk about it

Reply:
Cool - let's put some time aside for a chat about it tomorrow then. I could do just before or after the morning tea?

Reply:
I've gone with inheritance. Looking forward to chatting over the pros and cons after morning tea though